### ISSUE TITLE
Cannot config with `host`

### ISSUE THREAD

**Issue by lasseebert** [larskrantz/logger_papertrail_backend#9]
After upgrading to 0.2.0, the following config fails, although it is still stated in the readme as a valid config:

```
config :logger, :logger_papertrail_backend,
  host: "somehost:someport",
  level: :info,
  system_name: "my_app"
```

Changing to `url` fixes it:

```
config :logger, :logger_papertrail_backend,
  url: "papertrail://somehost:someport/my_app",
  level: :info
```

**Comment by lasseebert** [comment]
This is the error I get when trying with the first config:

```
** (Mix) Could not start application logger: Logger.App.start(:normal, []) returned an error: shutdown: failed to start child: Logger.Watcher
    ** (EXIT) an exception was raised:
        ** (MatchError) no match of right hand side value: {:error, {{{:case_clause, {:EXIT, {%LoggerPapertrailBackend.ConfigurationError{message: "Unknown configuration: [host: \"<HOST>:<PORT>\", level: :info, system_name: \"legolas development\", format: \"$metadata $message\"]"}, [{LoggerPapertrailBackend.Configurator, :configure_papertrail_target, 1, [file: 'lib/configurator.ex', line: 56]}, {LoggerPapertrailBackend.Logger, :configure, 2, [file: 'lib/logger_papertrail_backend/logger.ex', line: 63]}, {LoggerPapertrailBackend.Logger, :init, 1, [file: 'lib/logger_papertrail_backend/logger.ex', line: 19]}, {:gen_event, :server_add_handler, 4, [file: 'gen_event.erl', line: 429]}, {:gen_event, :handle_msg, 5, [file: 'gen_event.erl', line: 274]}, {:proc_lib, :init_p_do_apply, 3, [file: 'proc_lib.erl', line: 247]}]}}}, [{Logger.Watcher, :do_init, 3, [file: 'lib/logger/watcher.ex', line: 78]}, {:gen_server, :init_it, 6, [file: 'gen_server.erl', line: 328]}, {:proc_lib, :init_p_do_apply, 3, [file: 'proc_lib.erl', line: 247]}]}, {:child, :undefined, {Logger.Watcher, {Logger, LoggerPapertrailBackend.Logger}}, {Logger.Watcher, :watcher, [Logger, LoggerPapertrailBackend.Logger, LoggerPapertrailBackend.Logger]}, :transient, 5000, :worker, [Logger.Watcher]}}}
            (logger) lib/logger/watcher.ex:16: anonymous fn/2 in Logger.Watcher.start_link/3
            (elixir) lib/enum.ex:1755: Enum."-reduce/3-lists^foldl/2-0-"/3
            (logger) lib/logger/watcher.ex:15: Logger.Watcher.start_link/3
            (stdlib) supervisor.erl:365: :supervisor.do_start_child/2
            (stdlib) supervisor.erl:348: :supervisor.start_children/3
            (stdlib) supervisor.erl:314: :supervisor.init_children/2
            (stdlib) gen_server.erl:328: :gen_server.init_it/6
            (stdlib) proc_lib.erl:247: :proc_lib.init_p_do_apply/3
```

**Comment by larskrantz** [comment]
Thanks for reporting, I'm on it, I can already see some brainfart-code in Configurator..

**Comment by larskrantz** [comment]
Published 0.2.1, closing #9

**Comment by lasseebert** [comment]
Very nice. Thanks for the quick fix :+1:
